KerbalStuff/objects.py

```diff
@@ -247,10 +247,10 @@ def __repr__(self) -> str:
 class DownloadEvent(Base):  # type: ignore
     __tablename__ = 'downloadevent'
     id = Column(Integer, primary_key=True)
-    mod_id = Column(Integer, ForeignKey('mod.id'))
+    mod_id = Column(Integer, ForeignKey('mod.id'), index=True)
```

Comment:
It doesn't match the migration script like this. The next time someone runs `alembic migrate`/`./generate_revision.sh` it will create something messy.

We could declare the indices like this instead, if I'm not mistaken:
```python
class DownloadEvent(Base):  # type: ignore
...
    Index('ix_downloadevent_mod_id_created', mod_id, created)
    Index('ix_downloadevent_version_id_created', version_id, created)
```

Reply:
Are you sure it can be included within the class declaration? All of the examples that I've found show it as a top-level command after the class is closed.

Reply:
Yeah, I've also seen it done differently mostly, outside the class. But one comment on a SO answer had it like that, and it _does_ appear to work, Alembic doesn't generate a new/different migration script.

Reply:
OK, moved it into the class since it does look much nicer that way.